src/test/unit/parsers/test_parser.py

```diff
@@ -45,7 +45,7 @@ def test_apigw_parse_response():
             "host": "dummy",
             "response": {
                 "headers": '{"apigw-requestid": "LY_66j0dPHcESCg="}',
-                "body": "",
+                "body": '""',
```

Comment:
Think that empty should stay `""` and not `'""'`, Better to send `null` I think if there is no-body, but second choice is `""`

Reply:
Note that `""` is not a valid json - it's actually nothing (the quotes are pythonics).
There is either:
1. An empty string, which is is `""` (so in python it will look like `'""'`)
2. Or null, which is `null` (so in python it will look like `"null"`)
And they represent different values...